kss/builder/decanter/kss-assets/css/kss.css

```diff
@@ -867,7 +867,7 @@ body {
   -webkit-box-direction: normal;
       -ms-flex-direction: column;
           flex-direction: column; }
-  body > .global-footer {
+  body > footer {
```

Comment:
We cannot assume that the element we want to target will be a direct descendant of the body tag. See comments in https://github.com/SU-SWS/decanter/issues/298